.github/workflows/simnode.yml

```diff
@@ -22,12 +22,16 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-          ref: ${{ github.event.pull_request.head.sha }}  
+
+      - name: Get branch name
+        id: branch-name
+        uses: tj-actions/branch-names@v5.1
 
       - name: Check for runtime changes
         id: check_runtime
         env:
           BASE_BRANCH: ${{ github.event.pull_request.base.ref }}
+          GITHUB_BRANCH_NAME: ${{ steps.branch-name.outputs.current_branch }}
```

Comment:
we also need thiis forr `.maintain/run_simnode.sh`

Reply:
and benchmarks

Reply:
Updated